.travis.yml

```diff
@@ -1,4 +1,3 @@
 sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: PATH="./bats/bin:$PATH" test/run
```

Comment:
You're welcome - thanks for the kind words!

Regarding the issue - I don't know what's going on there. Could it be something that bats is doing? The temp folder structure looks odd.